yarn.lock

```diff
@@ -18740,6 +18740,11 @@ shell-escape@^0.2.0:
   resolved "https://registry.yarnpkg.com/shell-escape/-/shell-escape-0.2.0.tgz#68fd025eb0490b4f567a027f0bf22480b5f84133"
   integrity sha1-aP0CXrBJC09WegJ/C/IkgLX4QTM=
 
+shell-escape@^0.2.0:
```

Comment:
I believe this is a mistake? could you revert this file?

Reply:
Done. I'm not sure why it changed.. possibly [#4379](https://github.com/yarnpkg/yarn/issues/4379)?